src/qmapshack/gis/rte/router/CRouterRoutino.cpp

```diff
@@ -248,19 +245,49 @@ void CRouterRoutino::buildDatabaseList()
                 continue;
             }
 
+            // qDebug() << "buildDatabase Prefix" << prefix;
+
 #ifdef Q_OS_WIN
             Routino_Database * data = Routino_LoadDatabase(dir.absolutePath().toLocal8Bit(), prefix.toLocal8Bit());
 #else
             Routino_Database * data = Routino_LoadDatabase(dir.absolutePath().toUtf8(), prefix.toUtf8());
 #endif
-            if(data)
+            qDebug() << "Loaded Routino DB" << dir.absolutePath().toUtf8().data() << "  " << prefix.toUtf8().data();
+
+            if(data == nullptr)
             {
-                comboDatabase->addItem(prefix.replace("_", " "), quint64(data));
+                QMessageBox::critical(this, "Routino Load...", xlateRoutinoError(Routino_errno), QMessageBox::Abort);
```

Comment:
Missing tr macro `Routino Load ...`. This has to be translated.

Reply:
Thanks missed that.  Changed it back to just "Routino ..."  which is used is other places.  So no tr() required